src/py/flwr/server/strategy/bulyan_test.py

```diff
@@ -0,0 +1,241 @@
+# Copyright 2020 Adap GmbH. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+# ==============================================================================
+"""Bulyan tests."""
+
+
+from typing import List, Tuple
+from unittest.mock import MagicMock
+
+from numpy import array, float32
+
+from flwr.common import (
+    Code,
+    FitRes,
+    NDArrays,
+    Parameters,
+    Status,
+    ndarrays_to_parameters,
+    parameters_to_ndarrays,
+)
+from flwr.server.client_proxy import ClientProxy
+from flwr.server.fleet.grpc_bidi.grpc_client_proxy import GrpcClientProxy
+
+from .bulyan import Bulyan
+
+
+def test_krum_num_fit_clients_20_available() -> None:
+    """Test num_fit_clients function."""
+    # Prepare
+    strategy = Bulyan()
+    expected = 20
+
+    # Execute
+    actual, _ = strategy.num_fit_clients(num_available_clients=20)
+
+    # Assert
+    assert expected == actual
```

Comment:
Let's remove test like this if they only test functionality not modified in the `Bulyan` strategy.